refinery/core/migrations/0001_initial.py

```diff
@@ -15,6 +15,7 @@ class Migration(migrations.Migration):
         ('data_set_manager', '0001_initial'),
         ('auth', '0001_initial'),
         ('file_store', '0001_initial'),
+        ('registration', '0001_initial')
```

Comment:
I haven't had much time to think about this yet, but it still seems scary

Reply:
Yes. Perhaps, we can discuss in dev. Feel free to mess around with this branch locally if you want to try anything.